src/libsystemd/sd-bus/bus-match.c

```diff
@@ -997,19 +997,19 @@ const char* bus_match_node_type_to_string(enum bus_match_node_type t, char buf[]
                 return "path_namespace";
 
         case BUS_MATCH_ARG ... BUS_MATCH_ARG_LAST:
-                snprintf(buf, l, "arg%i", t - BUS_MATCH_ARG);
+                (void) snprintf(buf, l, "arg%i", t - BUS_MATCH_ARG);
                 return buf;
 
         case BUS_MATCH_ARG_PATH ... BUS_MATCH_ARG_PATH_LAST:
-                snprintf(buf, l, "arg%ipath", t - BUS_MATCH_ARG_PATH);
+                (void) snprintf(buf, l, "arg%ipath", t - BUS_MATCH_ARG_PATH);
                 return buf;
 
         case BUS_MATCH_ARG_NAMESPACE ... BUS_MATCH_ARG_NAMESPACE_LAST:
-                snprintf(buf, l, "arg%inamespace", t - BUS_MATCH_ARG_NAMESPACE);
+                (void) snprintf(buf, l, "arg%inamespace", t - BUS_MATCH_ARG_NAMESPACE);
                 return buf;
 
         case BUS_MATCH_ARG_HAS ... BUS_MATCH_ARG_HAS_LAST:
-                snprintf(buf, l, "arg%ihas", t - BUS_MATCH_ARG_HAS);
+                (void) snprintf(buf, l, "arg%ihas", t - BUS_MATCH_ARG_HAS);
```

Comment:
Ignoring the return value here seems wrong…

Reply:
moved to snprintf_ok